tests/search.service.spec.js

```diff
@@ -44,7 +44,7 @@ describe("Searching for hackers", function () {
             .end(function (err, res) {
                 res.should.have.status(200);
                 res.body.should.have.property('data');
-                res.body.data.should.have.length(1);
+                res.body.data.should.have.length(2);
```

Comment:
Why does this have length 2 now?

Reply:
I made a new Hacker with status 'Waitlisted', as the previous two Hackers had statuses 'Accepted' and 'Confirmed' respectively. This Hacker happened to be female.

Reply:
Sounds good